Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -47,12 +47,21 @@ kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files
 
 ## Deploying a second cluster
 
-If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a single operator to manage both clusters execute the following:
+If you wish to create a new CephCluster in a different namespace than `rook-ceph` while using a
+single operator to manage both clusters change `SECOND_ROOK_CLUSTER_NAMESPACE`. The `common-second-cluster`
+manifests is commented to allow for easy `sed` replacements.
 
 ```console
 cd deploy/examples
+export ROOK_OPERATOR_NAMESPACE="rook-ceph"
+export SECOND_ROOK_CLUSTER_NAMESPACE="rook-ceph-secondary"
+
+sed -i.bak \
+    -e "s/\(.*\):.*# namespace:operator/\1: $ROOK_OPERATOR_NAMESPACE # namespace:operator/g" \
+    -e "s/\(.*\):.*# namespace:cluster/\1: $SECOND_ROOK_CLUSTER_NAMESPACE # namespace:cluster/g" \
+  common-second-cluster.yaml
 
-NAMESPACE=rook-ceph-secondary envsubst < common-second-cluster.yaml | kubectl create -f -
```

Comment:
I like getting rid of this b/c:

1. more consistency across docs/code
2. allows us to use the manifest with default values for quicker dev work